src/sulfuras.js

```diff
@@ -1,5 +1,10 @@
 class Sulfuras {
-    constructor(name, sellIn, quality) {
+    constructor(name, sellIn, quality){
+        const qualityError = "Quality must be within 0 - 50 inclusive.";
```

Comment:
Defining a `QualityError` type will also mean you don't have to define this const three times.